.gitattributes

```diff
@@ -0,0 +1 @@
+iris/docs/archive/*.zip filter=lfs diff=lfs merge=lfs -text
```

Comment:
This conforms to the correct format, as far as I can tell ([link](https://rehansaeed.com/gitattributes-best-practices/#git-large-file-system-lfs)).